integration_tests/test_real_sdk.py

```diff
@@ -243,19 +247,35 @@ def test_real_sdk_models_optimal_planar_processing_changes_signal(model_type):
     key = os.environ["AIC_SDK_LICENSE"]
     model_enum = AICModelType(model_type)
 
-    with Model(model_enum, license_key=key, sample_rate=48000) as m:
+    # STT models have specific optimal sample rates (8kHz or 16kHz), not 48kHz
+    # Determine probe sample rate based on model type
+    is_stt_model = model_type in (8, 9, 10, 11, 12)
```

Comment:
Isn't it possible to use the actual model names here rather than these constants?